### PR TITLE
update to AI nuget 1.0, fix persistenceChannelreference

### PR DESCRIPTION
**Pull request by SoubhagyaDash**
update to AI nuget 1.0, fix persistenceChannelreference

### CODE REVIEW

File: Samples/AzureEmailService/MvcWebRole/ApplicationInsights.config [hunk 1]
```diff
@@ -33,6 +33,8 @@
       You may provide an optional ReportAs attribute which will be used as the metric name when reporting counter data.
       For the purposes of reporting, metric names will be sanitized by removing all invalid characters from the resulting metric name.
       
+      NOTE: performance counters configuration will be lost upon NuGet upgrade.
```
Comment (SergKanz) [inline Samples/AzureEmailService/MvcWebRole/ApplicationInsights.config]: do you need this note?

File: Samples/AzureEmailService/WorkerRoleA/ApplicationInsights.config [hunk 2]
```diff
@@ -22,46 +33,55 @@
...
-      -->
-    </Add>
-    <Add Type="Microsoft.ApplicationInsights.Extensibility.Web.TelemetryInitializers.WebSessionTelemetryInitializer, Microsoft.ApplicationInsights.Extensibility.Web"/>    
-  </TelemetryInitializers>
+<!-- 
+    Learn more about Application Insights configuration with ApplicationInsights.config here: 
+    http://go.microsoft.com/fwlink/?LinkID=513840
+    
+    Note: If not present, please add <InstrumentationKey>Your Key</InstrumentationKey> to the top of this file.
+  --><!--
```
Comment (SergKanz) [inline Samples/AzureEmailService/WorkerRoleA/ApplicationInsights.config]: clean up comments. You don't need duplicated comments here

File: Samples/AzureEmailService/MvcWebRole/ApplicationInsights.config (hunk 1 above)
Comment (SoubhagyaDash) [inline Samples/AzureEmailService/MvcWebRole/ApplicationInsights.config]: This is added by the perf.counter nuget - should remain here.

**Comment by SergKanz** [comment]
Looks better. Please clean up comments before pushing to master

**Comment by SergKanz** [comment]
:shipit:

**Pull request merged**